src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
```

Comment:
it is better to remove empty lines inside of this method. As the logic is simple and so it should not be separated with empty lines.